assets/events.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+
+import simplejson as json
+import requests
+import logging as log
+import os
+import sys
+
+
+class EventsResource:
+    """
+        Events resource implementation.
+        This python script is the target of symbolic links and is used for check, in and out.
+        These three commands are defined as methods on this class and common parameters live in the constructor.
+        To enable the debug output, the resource source configuration must have the debug parameter.
+    """
+
+    def __init__(self, command_name, json_data, command_argument):
+        self.command_name = command_name
+        self.command_argument = command_argument
+        # Namespace the approval lock
+        self.data = json.loads(json_data)
+        self.token = None
+
+        self.api_url = None
+        self.api_login = None
+        self.api_password = None
+        self.organization = None
+        self.icon = None
+        self.messagel = None
+        self.severity = None
+        self.tags = None
+        self.title = None
+        self.type = None
+
+        # allow debug logging to console for tests
+        if os.getenv('RESOURCE_DEBUG', False) or self.data.get('source', {}).get('debug', False):
+            log.basicConfig(level=log.DEBUG)
+        stderr = log.StreamHandler()
+        stderr.setLevel(log.INFO)
+        log.getLogger().addHandler(stderr)
+
+        log.debug('command: %s', command_name)
+        log.debug('input: %s', self.data)
+        log.debug('args: %s', command_argument)
+        log.debug('environment: %s', os.environ)
+
+
+    def check_cmd(self, source, version):
+        """
+        Check for new version(s)
+
+        #TODO this first release of the resource won't check for new events/won't trigger
+        But in a future version, we may want to check/filter specific events to trigger the pipeline
+
+        :param source: is an arbitrary JSON object which specifies the location of the resource,
+        including any credentials. This is passed verbatim from the pipeline configuration.
+        :param version: is a JSON object with string fields, used to uniquely identify an instance of the resource.
+        :return: a dict with the version fetched
+        """
+
+        log.debug('version: %s', version)
+
+        if not version:
+            version = {"timestamp": '0'}
+        log.debug('source: %s', source)
+        log.debug('version: %s', version)
+
+        return versions_list
```

Comment:
This variable is undeclared, but I guess the method wasn't implemented yet. Maybe removing its content would be more explicit with the issue?